deployments/fargate/README.md

```diff
@@ -0,0 +1,137 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, it is assumed that the
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS Task in a **sidecar** container alongside monitored
+applications.
+
+Applies to Collector release v0.30.0 and above which corresponds to image tag 0.30.0 and above
+in the image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+Overview of the container definition for the Collector:
```

Comment:
It seems like this is a combination of 1) configuration options 2) explanation of what things are -- I would separate these things. In addition, I would put the getting started information as close to the top as possible.

Reply:
Ok